docs/api.md

```diff
@@ -49,6 +49,33 @@ function after_enqueue_script( array $args ) {
 }
 ```
 
+### `update_vary_headers`
+
+Adds header key and custom data to vary header
```

Comment:
I think it's worth being a bit more verbose about the intended use case of this function because it's still a little vague.